lib.md

```diff
@@ -0,0 +1,761 @@
+## Usage
```

Comment:
Is this old before files moved to docs/?

Reply:
deleted this my bad. Were quite a few files on the way to using docs dir.